packages/guides/integrating-the-safe-core-sdk.md

```diff
@@ -0,0 +1,353 @@
+  transactionHash: string
+  safeTxHash: string
+  executor?: string
+  isExecuted: boolean
+  isSuccessful?: boolean
+  ethGasPrice?: string
+  gasUsed?: number
+  fee?: number
+  origin: string
+  dataDecoded?: string
+  confirmationsRequired: number
+  confirmations?: [
+    {
+      owner: string
+      submissionDate: string
+      transactionHash?: string
+      confirmationType?: string
+      signature: string
+      signatureType?: string
+    },
+    // ...
+  ]
+  signatures?: string
+}
+```
+
+## 7. Confirm/reject the transaction <a name="confirm-transaction"></a>
+
+The owners of the Safe can now sign the transaction obtained from the Safe Transaction Service by calling the method `signTransactionHash` from the Safe Core SDK to generate the signature and by calling the method `confirmTransaction` from the Safe Service Client to add the signature to the service.
+
+```js
+// transaction: SafeMultisigTransactionResponse
+
+const hash = transaction.safeTxHash
+let signature = await safeSdk.signTransactionHash(hash)
+await safeService.confirmTransaction(hash, signature.data)
+```
+
+## 8. Execute the transaction <a name="execute-transaction"></a>
+
+Once there are enough confirmations in the service the transaction is ready to be executed. The account that will execute the transaction needs to retrieve it from the service with all the required signatures and call the method `executeTransaction` from the Safe Core SDK.
+
+The method `executeTransaction` expects an instance of the class `SafeTransaction` so the transaction needs to be transformed from the type `SafeMultisigTransactionResponse`.
+
+```js
+import { EthSignSignature } from '@gnosis.pm/safe-core-sdk'
+
+// transaction: SafeMultisigTransactionResponse
+
+const safeTransactionData: SafeTransactionData = {
+  to: transaction.to,
+  value: transaction.value,
+  data: transaction.data,
+  operation: transaction.operation,
+  safeTxGas: transaction.safeTxGas,
+  baseGas: transaction.baseGas,
+  gasPrice: transaction.gasPrice,
+  gasToken: transaction.gasToken,
+  refundReceiver: transaction.refundReceiver,
+  nonce: transaction.nonce
+}
+const safeTransaction = await safeSdk.createTransaction(safeTransactionData)
+transaction.confirmations.forEach(confirmation => {
+  const signature = new EthSignSignature(confirmation.owner, confirmation.signature)
+  safeTransaction.addSignature(signature)
+})
+
+const executeTxResponse = await safeSdk.executeTransaction(safeTransaction)
+const receipt = executeTxResponse.transactionResponse && (await executeTxResponse.transactionResponse.wait())
+```
+
+## 9. Interface checks <a name="interface-checks"></a>
+
+During the process of collecting the signatures and executing the transaction some useful checks can be used in the interface to display or hide a button to confirm or execute the transaction depending on the current number of confirmations, the address of the accounts who confirmed the transaction and the Safe threshold:
```

Comment:
"...collecting signatures/executing transactions, some useful checks can be made in the interface..."

Reply:
"...address of accounts that confirmed..."